bin/bin.js

```diff
@@ -287,6 +293,22 @@ const main = async () => {
       }
     }
 
+    if (exec) {
+      const filenameBase = episodeFilename.substring(
+        0,
+        episodeFilename.lastIndexOf(".")
+      );
+      const execCmd = exec
+        .replace("{}", `"${outputPodcastPath}"`)
+        .replace("{filenameBase}", `"${filenameBase}"`);
+      try {
+        child.execSync(execCmd, { stdio: "ignore" });
```

Comment:
Currently no logging when exec runs successfully, only errors. Logging the whole output of e.g. `ffmpeg` is too much anyways. So I would stick to only logging errors. What do you think?